### ISSUE TITLE
Cannot run "jasmine examples"

### ISSUE THREAD

**Issue by walterdolce** [jasmine/jasmine#1165]
Hi, I was just starting up with `jasmine`, but I got this:
```
fs.js:951
  return binding.readdir(pathModule._makeLong(path), options.encoding);
                 ^

Error: ENOENT: no such file or directory, scandir '{{MY_PATH}}/node_modules/jasmine/node_modules/jasmine-core/lib/jasmine-core/example/node_example/spec/helpers/jasmine_examples'
    at Error (native)
    at Object.fs.readdirSync (fs.js:951:18)
    at copyFiles ({{MY_PATH}}/node_modules/jasmine/lib/command.js:186:24)
    at Object.installExamples [as action] ({{MY_PATH}}/node_modules/jasmine/lib/command.js:127:3)
    at Command.run ({{MY_PATH}}/node_modules/jasmine/lib/command.js:47:20)
    at Object.<anonymous> ({{MY_PATH}}/node_modules/jasmine/bin/jasmine.js:11:9)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
```

Not sure why it happens yet. You probably will be faster than me in understanding why the error is occurring.

**Comment by Gerg** [comment]
This looks like an issue with https://github.com/jasmine/jasmine-npm. @amavisca should be able to help.

**Comment by amavisca** [comment]
It's fixed on master with this commit: 073e3a78fd29fd3409d88518968da8aae5f24c37

**Comment by walterdolce** [comment]
Thanks all. I guess I should wait for the next release to happen.
